conferences/presentations/UKRI_cloud_2019-02-12/UKRI_cloud_demo_2019-02-12.md

```diff
@@ -0,0 +1,129 @@
+# Reproducible, Reliable, Reusable Analyses with BinderHub on Cloud
+
+Sarah Gibson, _The Alan Turing Institute_
+
+## What does "Reproducibility" mean?
+
+* It has a lot of different meanings across research fields
+* In this context, to be "reproducible" means the same results (e.g. that are published in a paper) are generated given the same input data pushed through the same analysis pipeline
+
+**An independent person should be able to easily check my (our) work.**
+
+## An Example of Not Being Reproducible
+**(AKA: I'm guilty of this too...)**
+
+<html><img src="graduation.png" alt="graduation" height="415" width="280"></html>
+
+* Astrophysics PhD, graduated January 2019
+* Researched phenomenon known as Gamma-Ray Bursts (not your average supernova...) and the neutron stars that power them
+
+<html><img src="figure2.png" alt="figure2" height="400" width="320"></html>
+
+* This is a figure published in [my first journal paper](https://arxiv.org/pdf/1706.04802.pdf)
+* It describes the evolution of the spin of a neutron star (middle panel) and the mass of its accretion disc (top panel) as it was being fed by fallback accretion
+* The bottom panel is the property we were interested in which is when a derived parameter would cross a specific threshold (the dashed line)
+
+**What my research was about isn't necessarily important.
+What is important is whether other scientists in my field could verify my work.**
+
+<html><img src="fig2_creation.gif" alt="fig2_creation"></html>
+
+This is a GIF of my PhD laptop producing the figure which turned out to **not** be reproducible for a number of reasons:
+* Was not version controlled
+* Computing environment(s) was not documented
+* Computing environment no longer exists - the laptop has been returned and wiped :scream:
+
+## Binder to the Rescue!
+
+With a little bit of work, I've managed to reproduce my figure using Binder (https://mybinder.org).
+
+* My code is in a public GitHub repo - now version controlled :ballot_box_with_check:
+* The computing environment has been documented in an `environment.yml` file :ballot_box_with_check: _(other config file types are available)_
+
+My PhD repo: https://bit.ly/2DacNJQ
```

Comment:
Why does this link go to a particular commit? Is it to make sure it always works?

I think its easy to link to the repo itself (see suggestion) but what's much longer and uglier is the link to the binder - can you add those as bit.ly links but describe what they are? You can also customise the bit.ly link so they don't just have random letters which would look tidier if you have time.

```suggestion
My PhD repo: https://github.com/sgibson91/magprop, binder: https://bit.ly/2DacNJQ
```

Reply:
This link goes to a particular commit to avoid my postBuild script which will make the binder load more slowly.

Reply:
Added my own version of this suggestion